src/helpers/BoxHelper.js

```diff
@@ -38,15 +39,15 @@ BoxHelper.prototype.update = ( function () {
 
 	return function update( object ) {
 
-		if ( object && object.isBox3 ) {
+		if ( object !== undefined ) {
 
-			box.copy( object );
+			console.warn( 'THREE.BoxHelper: .update() has no longer arguments.' );
 
-		} else {
+		}
 
-			box.setFromObject( object );
+		this.object.updateMatrixWorld( true );
```

Comment:
I've just realize that `Box3 .expandByObject()` calls `.updateMatrixWorld()`. I'll remove this line!